libres/lib/res_util/vector_conversion.cpp

```diff
@@ -0,0 +1,21 @@
+#include <vector>
+#include <string>
+#include <ert/util/stringlist.hpp>
+
+std::vector<std::string>
+stringlist_to_vector(const stringlist_type *string_list) {
+    std::vector<std::string> converted_vector;
+    for (int i = 0; i < stringlist_get_size(string_list); i++) {
+        std::string item = stringlist_iget(string_list, i);
+        converted_vector.push_back(item);
+    }
+    return converted_vector;
+}
+
+stringlist_type *vector_to_stringlist(const std::vector<std::string> &vector) {
```

Comment:
Is this meant to be a temporary solution until all usage of `stinglist_type` is eliminated?
If so, a comment would be nice.

Reply:
Yes, that was my intention, and can add a docstring. Will be happy to change it to a different solution if you can think of one though. This does help to make it a bit more contained.

Reply:
Did we agree to rewrite this PR such that `vector_to_stringlist` is not needed?